message_buffer/programs/message_buffer/src/lib.rs

```diff
@@ -84,6 +82,47 @@ pub mod message_buffer {
     ) -> Result<()> {
         instructions::put_all(ctx, base_account_key, messages)
     }
+
+
+    /// Initializes the buffer account with the target_size
+    ///
+    /// TODO: should target_size be a parameter or should
```

Comment:
@jayantk  what are your thoughts on this?

Reply:
target_size parameter is good. I think we have to manage the sizes anyway